requirements.txt

```diff
@@ -3,13 +3,15 @@ Pygments==2.18.0
 cohere==5.9.2
 configargparse
 fastapi==0.112.2
+google-ai-generativelanguage
```

Comment:
Could you also freeze the version? Since we're depending on quite a few external libraries it might lead to conflicts in the future (as we already did with `gradio` + `marqo`).